Python/ceval.c

```diff
@@ -127,6 +127,9 @@ lltrace_instruction(_PyInterpreterFrame *frame,
                     PyObject **stack_pointer,
                     _Py_CODEUNIT *next_instr)
 {
+    if (frame->owner == FRAME_OWNED_BY_CSTACK) {
```

Comment:
lltrace assumed that the `f_code` field is a code object, so it would crash on shim frames.
I've changed it to the safer `!PyCode_Check(frame->f_executable)` to support other kinds of executables.